### PR TITLE
Adding documentation for Class Settings to expose settings specific to Team's of type Class

### PR DESCRIPTION
**Pull request by gagarwal**
Adding documentation for Class Settings to expose settings specific to Teams' of type Class. For now, we are exposing a single Boolean value - notifyGuardiansAboutAssignments. This property will only be exposed if the Team type is of type Class.

### CODE REVIEW

**Comment by Lauragra** [comment]
@gagarwal , what is the merge status of this PR?

**Comment by gagarwal** [comment]
> @gagarwal , what is the merge status of this PR?

@Lauragra This is ready to be merged, but I do not have permission to update the label.

**Comment by Lauragra** [comment]
@gagarwal , please make sure that you follow the Get started steps in this topic to get contributor permission to the repo: https://msgo.azurewebsites.net/add/document/manage-content/get-started-with-github.html.

File: api-reference/beta/resources/team.md [hunk 1]
```diff
@@ -49,12 +49,13 @@ Every team is associated with a [group](../resources/group.md). The group has th
...
+|internalId | string | A unique ID for the team that has been used in a few places such as the audit log/[Office 365 Management Activity API](https://docs.microsoft.com/office/office-365-management-api/office-365-management-activity-api-reference). |
 |isArchived|Boolean|Whether this team is in read-only mode. |
 |memberSettings|[teamMemberSettings](teammembersettings.md) |Settings to configure whether members can perform certain actions, for example, create channels and add bots, in the team.|
 |messagingSettings|[teamMessagingSettings](teammessagingsettings.md) |Settings to configure messaging and mentions in the team.|
 |discoverySettings|[teamDiscoverySettings](teamdiscoverysettings.md) |Settings to configure team discoverability by others.|
 |webUrl|string (readonly) | A hyperlink that will go to the team in the Microsoft Teams client. This is the URL that you get when you right-click a team in the Microsoft Teams client and select **Get link to team**. This URL should be treated as an opaque blob, and not parsed. |
+|classSettings|[teamClassSettings](teamclasssettings.md) |Configure settings of a class. Available only when the team represents a *Class*.|
```
Comment (Lauragra) [inline api-reference/beta/resources/team.md]: Is this a reference to the educationClass resource?

Comment (gagarwal) [reply api-reference/beta/resources/team.md]: > Is this a reference to the educationClass resource?

Not sure about "educationClass" you are mentioning here?

**Pull request merged**